src/common/components/hosts/Hostname.tsx

```diff
@@ -53,9 +55,18 @@ const Hostname: React.FC<HostnameProps> = ({
 
   return onEditHostname ? (
```

Comment:
this can be simplified to
```suggestion
  return !readonly || onEditHostname ? (
```

no need for the other changes

Reply:
Here if I'm doing "!readonly || onEditHostname" so if there's a EditHostname but it's readonly it will show me the modal button. 
I checked it on my installed cluster.

Reply:
right, it should be 
```js
!readonly && onEditHostname ?
```

Reply:
Yes, that's what I thought. Thank you!